icu_benchmarks/data/pooling.py

```
@@ -101,15 +102,15 @@ def _save_pooled_data(self, data_dir, data, datasets, file_names, samples=10000)
     def _pool_datasets(
-            self,
-            datasets={},
-            samples=10000,
-            vars=[],
-            seed=42,
-            shuffle=True,
-            runmode=RunMode.classification,
-            data_dir=Path("data"),
-            save_test=True,
+        self,
+        datasets={},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)